### DISCUSSION TITLE
I lost access to my account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132940] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I have an existing account where all my clients' projects are stored (private repo). I've lost my security code and forgot to set up my phone number for 2FA OTP. I know my password to access the account but I'm stuck because of the security code.

Please, is there another way to resolve this?

**Comment by davevad93** [comment, marked as answer]
Hi @nisaps ,

If you have lost your 2FA credentials and the recovery codes as well, unfortunately you are permanently locked out of your account as stated in the [docs](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials), you can only unlink the email address associated to that account and use it for another account ([Unlinking your email address from a locked account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account)) but if you don't have any local copies of your private repos you won't be able to recover them.
